device/bgp/routepathattributes.yaml

```diff
@@ -22,3 +22,11 @@ components:
               x-field-uid: 1
             egp:
               x-field-uid: 2
+            incomplete:
+              x-field-uid: 3
+        local_preference:
+          description: >-
+            Local Preference attribute, for iBGP peers to select the path for the traffic leaving the AS.
```

Comment:
a) ... when multiple paths are available to the destination .
b) We can specify if larger value means higher preference or lower preference. 
c) I think we need an include_local_preference to allow user to disable inclusion of the attribute ( default value should be true ) esp. for eBGP use-case.

Reply:
updated,added include-local-pref and updated the descriptions